design-assets/README.md

```diff
@@ -0,0 +1,17 @@
+# CMSDS Design Assets
+
+In this folder you will find the following
+
+- CMSDS Sketch file
+- CMSDS fonts (Bitter, Icons, and Open Sans)
+
+To update the Sketch library version create a pull request with the following: 
+
+1. Update the CMSDS Sketch file
+1. Update the .xml file in the following ways
```

Comment:
Maybe we should include context on when the Sketch library is updated? I was under the impression the xml file would be updated when we do releases, this way we can include sketch changes in the release notes. This process seems to suggest to me that anyone can make a PR to update the sketch library at anytime

Reply:
Also, what do you think about keeping the sketch version and the npm package version the same? Aka starting off at 4.0.0

Reply:
I'm wondering if it would make sense to update the Sketch version if nothing in the Sketch file changed? For example, if we do a release is all technical bug that wouldn't have any impact on design assets.

Reply:
Currently thats the way our npm packages work, i.e. even if there was only changes in `core`, we publish `support` to the same version. That being said, this is only because of our monorepo setup, and the design assets don't have to follow this. 

The main case where I think it might make sense to version together is if the changelog for sketch files is combined with existing changelog we have on Github. This way a designer can easily reference a Sketch file's version changes on Github.

Reply:
Yeah, I think that makes sense - I was thinking if it would be confusing for Sketch users to see an update in their Sketch Library that does not contain anything other than a bump in the version number?

Reply:
@melwoodard I'd be curious about your thoughts here